charts/spire/charts/spire-server/values.yaml

```diff
@@ -445,38 +445,66 @@ controllerManager:
     - local-path-storage
 
   identities:
```

Comment:
Do we still need this extra `identities:` layer?

Reply:
Oh, yeah. I think we could get rid of that.... Good idea. Will take a stab at that.

Reply:
nm, i think its good to have it as it keeps everything clustered together in the docs.